packages/amplify-cli-core/API.md

```diff
@@ -1198,6 +1299,9 @@ export const isPackaged: boolean;
 // @public (undocumented)
 export const isResourceNameUnique: (category: string, resourceName: string, throwOnMatch?: boolean) => boolean;
 
+// @public (undocumented)
+export const isValidGraphQLAuthError: (message: string) => boolean;
```

Comment:
does this belong to cli-core ?

Reply:
https://github.com/aws-amplify/amplify-cli/pull/12069#discussion_r1117375855